desc/equilibrium/equilibrium.py

```diff
@@ -764,6 +826,87 @@ def get_axis(self):
         axis = FourierRZCurve(R_n, Z_n, modes_R, modes_Z, NFP=self.NFP, sym=self.sym)
         return axis
 
+    def set_poincare_equilibrium(self, zeta=0):
```

Comment:
is this needed?

Reply:
I think this should now happen automatically during init right?

Reply:
Yes, if you create a PoincareSurface() object, you can directly create a Poincare compatible equilibrium. But if you mean, "you updated every example file with the new script, so they should have proper xsxection. Creating additional equilibrium is unnecessary!", you are correct.

Reply:
One advantage of this function is that it returns a copy, but yeah for the final version, we can get rid of this. It helped for setting xsection.isgiven stuff (that we don't have anymore)